__tests__/index-test.js

```diff
@@ -0,0 +1,7 @@
+import * as Carbon from '../index';
+
+describe('Carbon Components React', () => {
+  it('can be imported', () => {
```

Comment:
Since the test can only fail based on path, can you change the `it` description to something like: 
`it('can be imported using the correct path')` just so that this is more specific.